src/get_webhook_data.py

```
@@ -66,6 +67,16 @@ class InputRequest(BaseModel):
 ## Variables
 
 token = os.environ.get("GIT_TOKEN")
+if not token:
+    try:
+        secret_id = "Daily_Range_GITHUB_TOKEN"
```

Ruff (0.8.2)

Possible hardcoded password assigned to: "secret_id"

(S105)